### ISSUE TITLE
Text file busy

### ISSUE THREAD

**Issue by fred01** [JetBrains/teamcity-docker-minimal-agent#10]
Sometimes this image runs into problem mentioned here https://github.com/docker/docker/issues/9547

Also, mentioned there 'sync'-workaround solves problem in this image as well, at least for me. 

Changes are in /run-services.sh
```
  if [ -f $entry ]; then
      chmod +x $entry
+      sync
      $entry
  fi
```

and in run-ahent.sh (i'm think this one is what actually helps)
```
LOG_DIR=/opt/buildagent/logs

-chmod +x ${AGENT_DIST}/bin/*.sh; check
+chmod +x ${AGENT_DIST}/bin/*.sh; check; sync

rm -f ${LOG_DIR}/*.pid
```

Can you add this fixes? Or can i add PR for this?

**Comment by VladRassokhin** [comment]
Thanks for noting, @fred01.
Fix in `run-services.sh` is already done in 1c532fc4
For `run-agent.sh` - will do the same

**Comment by VladRassokhin** [comment]
Would be included into next images: eap and 2017.1
